x-pack/plugins/security_solution/public/common/components/link_to/redirect_to_case.tsx

```diff
@@ -31,4 +31,4 @@ export const getCreateCaseUrl = (search?: string | null) =>
   `/create${appendSearch(search ?? undefined)}`;
 
 export const getConfigureCasesUrl = (search?: string) =>
-  `/configure${appendSearch(search ?? undefined)}`;
+  `/settings${appendSearch(search ?? undefined)}`;
```

Comment:
Don't forget to revert this!

Reply:
done in [fe265f6](https://github.com/elastic/kibana/pull/169532/commits/fe265f6c7737875b813e118b5f3b4449fb1b9571)